docs/src/lenses.md

```diff
@@ -26,6 +27,24 @@ julia> modify(lowercase, obj, lens)
 T("aa", "BB")
 ```
 
+Lenses can also be constructed directly and composed with [`opcompose`](@ref), `⨟`, or `∘` (note reverse order).
+
+```jldoctest
+julia> using Accessors
+
+julia> v = (a = 1:3, )
+(a = 1:3,)
+
+julia> l = opcompose(PropertyLens(:a), IndexLens(1))
```

Comment:
Maybe add a line like:
```
julia> opcompose(PropertyLens(:a), IndexLens(1)) === @optic _.a[1]
true
```
To demonstrate equivalence of the explicit form with the macro.

Reply:
done, thanks for all the suggestions!